package.json

```diff
@@ -53,8 +53,10 @@
     "ivygate": "https://github.com/kipr/ivygate#v0.1.0",
     "morgan": "^1.10.0",
     "node": "13.12.0",
+    "prop-types": "^15.8.1",
```

Comment:
Do we use this dependency?

Reply:
It's used by react-loading, the library I was using for the loading animations